docs/migration.md

```diff
@@ -842,33 +843,17 @@ extensions:
     time_zone: America/Tijuana
 
 receivers:
-  ## Use TCP receiver for TCP protocol
-  tcplog/first receiver:
-    ## listen address in format host:port
-    ## host 0.0.0.0 mean all network interfaces
-    listen_address: 0.0.0.0:514
-    ## Add network attributes
-    ## `net.peer.name` is going to be used as processors.source.source_host
-    ## rel: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.72.0/receiver/tcplogreceiver#configuration
-    add_attributes: true
-  ## Use UDP receiver for UDP protocol
-  udplog/first receiver:
-    ## listen address in format host:port
-    ## host 0.0.0.0 mean all network interfaces
-    listen_address: 0.0.0.0:514
-    ## Add network attributes
-    ## `net.peer.name` is going to be used as processors.source.source_host
-    ## rel: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.72.0/receiver/udplogreceiver#configuration
-    add_attributes: true
+  ## Use a syslog receiver
+  syslog:
+    tcp:
+      listen_address: "0.0.0.0:54526"
+    protocol: rfc5424
+    operators:
+      - type: syslog_parser
+        protocol: rfc5424
```

Comment:
We have been using tcplog, because it doesn't verify correctness of syslog format. Like Installed Collector. Installed Collector's syslog is more like tcp/udp log receiver that syslogreceiver

Reply:
The issue here is that we want to behave as close as possible to Installed Collector, so I wouldn't try to follow the rfc in this document. We don't want to change behavior for customers.

To be honest, Syslog source in Installed Collector should be named TCP Source, but it is what it is :shrug:

Reply:
Also we are using sumologic exporter for all of data in this document. If we use `syslog_parser` the customers will get different data that now, as now they get raw syslog lines

Reply:
so I tested this and it looks like the pipeline will not work without the `syslog_parser` operator at the moment

Reply:
btw, we should fix syslog exporter to no send
`%!d(<nil>) 1970-01-01T00:00:00Z %!s(<nil>) %!s(<nil>) %!s(<nil>) %!s(<nil>) - %!s(<nil>)`